README.rst

```diff
@@ -2,51 +2,122 @@
+    :target: https://github.com/opinionated-digital-center/python-library-project-generator/actions?query=workflow%3A%22Test+GitLab+Pipeline%22
+
+* GitHub repo: https://github.com/opinionated-digital-center/python-library-project-generator .
+* Free software: MIT license.
+
+The "Opinionated Python Library Project Template" is a fully tested, feature rich
+`Cookiecutter`_ scaffold to kick start your Python library and command line interface
+development.
+
+It is "opinionated" as it reflects strong structural and tooling choices,
+which we believe help developers being more effective within Digital Centers. It can
+however seamlessly be used for Open Source development (works also out-of-the-box with
+gitlab.com, github.com, pypi.org and readthedocs.org, as a standalone project).
+
+In addition to the intrinsic features offered by the template (see "Features list"
+below), the specific features that make it suitable and effective for Digital Centers
+are the abilities to:
+
+* Use self-hosted GitLab and GitHub.
+* Publish to self-hosted package repositories.
+* Reuse centralised and optimised pipeline elements. (available soon)
+* Reuse other centralised "toolbox" items.
+
+The argumentation behind the Opinionated Digital Center's choices (for this and other
+projects) are/will be gradually documented as Architecture Decision Records, either
+on `ODC's ADR repository <https://github.com/opinionated-digital-center/architecture-decision-record>`_
+for transversal decisions, or on each individual repository's ``docs/adr`` directory
+(`here <https://github.com/opinionated-digital-center/python-library-project-generator/tree/master/docs/adr>`_
+for this project) for project related decisions.
+
+
+Fully tested features
+---------------------
+All the features of this template are fully tested through CI/CD pipelines, including:
+
+* The `Test and make release <https://github.com/opinionated-digital-center/python-library-project-generator/actions?query=workflow%3A%22Test+and+make+release%22>`_
+  pipeline to:
+
+  * Generate projects for each options of the template.
+  * Run the project's built-in tests and checks for each option specifically.
+  * Upon success, release a new version of the template.
+
+* The `Test GitLab Pipeline <https://github.com/opinionated-digital-center/python-library-project-generator/actions?query=workflow%3A%22Test+GitLab+Pipeline%22>`_
+  pipeline (GitHub available soon) to:
+
+  * Generate a project with the default options.
+  * Test that the generated project's GitLab pipeline runs successfully, including:
+
+    * Run the project's built-in tests and checks on the project's pipeline.
+    * Automatically bump the package's version following `Semantic Versioning`_ (based
+      on commit messages following the `Angular Commit Message Guideline`_, a subset
+      of the `Conventional Commits specification`_).
```

Comment:
Maybe we could simplify this by pointing to the corresponding ADR

Reply:
done